bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/EdCompTargetList.java

```diff
@@ -110,6 +111,10 @@ public void keyPressed(KeyEvent event) {
 
         _agsPub.subscribe((obs, oldOptions, newOptions) -> updateGuiding());
 
+        BagsManager.instance().addBagsStatusListener((obs, oldStatus, newStatus) -> {
+            if (obs == getContextObservation())
```

Comment:
I think here we actually DO want the references to be the same, rather than do an in-depth comparison of the objects. The in-depth comparison would be costly, and it only applies if the references are the same.